src/desktop/space/element.rs

```diff
@@ -10,6 +10,33 @@ use std::{
 };
 use wayland_server::protocol::wl_surface::WlSurface;
 
+/// Indicates default values for some zindexs inside smithay
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[repr(u8)]
+pub enum RenderZindex {
+    /// WlrLayer::Background default zindex
+    Background = 10,
+    /// WlrLayer::Bottom default zindex
+    Bottom = 20,
+    /// Default zindex for Windows
+    Shell = 30,
+    /// Default zindex for Windows PopUps
+    PopUpsShell = 40,
```

Comment:
Please just rename to `Popups` and move after `Top` (the top-layer can also have popups + popups should also be rendered before the top-layer).